### PR TITLE
fix: enable RLS for tables

### PR DESCRIPTION
**Pull request by gentlementlegen**
Added RLS security for tables to avoid having the data readable / writable by anyone.

### CODE REVIEW

**Comment by gentlementlegen** [comment]
@sshivaditya2019 rfc

**Review by 0x4007 — COMMENTED** [review]
I don't think this really matters much if the auth token is hidden.

**Comment by gentlementlegen** [comment]
That's true, it is more to avoid Supabase warnings and updated the `README` with it as well.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To set up the `.dev.vars` file, you will need to provide the following variables
 ## Usage
 - Add the following to your `.ubiquibot-config.yml` file with the appropriate URL:
 ```yaml
-- plugin: ubiquity-os-marketplace/generate-vector-embeddings
+- plugin: https://ubiquity-os-comment-vector-embeddings-main.ubiquity.workers.dev
```
Comment (0x4007) [inline README.md:15]: I just realized maybe for the workers URLs it makes sense to do this:

```suggestion
- plugin: https://comment-vector-embeddings.ubiquity-os.workers.dev
```

We now have a dedicated GitHub organization to host the plugins (@ubiquity-os-marketplace) so we may no longer need the `ubiquity-os-` prefix in the plugin name?

**Review by 0x4007 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gentlementlegen) [reply README.md:15]: Yes you can remove the `ubiquity-os` prefix. We have to figure out how to handle the domain url, let's keep discussing this on the other pull-request you opened.

**Pull request merged**